module/nbs/Player/src/Player.cpp

```diff
@@ -30,22 +30,24 @@
 
 #include "extension/Configuration.hpp"
 
-namespace module::nbs {
```

Comment:
this looks like you've rewritten the Player module? Seems like something for a separate PR?